<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T-Con Board Restore</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>576</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>T-CON Board Restore</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NAND</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EEPROM</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Verify</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Encoding</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_5">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckPID</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tcp_status_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0ms - Please Connect T-Con Zig</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Please Connect I2C Board</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Model Info</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Resolution</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Model</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>FHD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FHD_RO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QHD_RO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UHD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UHD_RO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>WUHD_RO</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Panel ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NAND</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EEPROM</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Elspsed Time - 00 :00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Status :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Status :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Data Restore</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="all_load">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ALL Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="all_write">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ALL Write</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
